### DISCUSSION TITLE
Getting error You exceeded your quota for the requested resource.

### DISCUSSION THREAD

**Discussion by mehrdadsamadie** [abuzuhri/Amazon-SP-API-CSharp#627] [Q&A]
Dose anyone know why I am getting this error "You exceeded your quota for the requested resource." when I am using GetOrderdetail

**Comment by TechyChap** [comment]
There's a maximum rate for each of the resources

https://developer-docs.amazon.com/sp-api/docs/usage-plans-and-rate-limits-in-the-sp-api
